src/custom/constants/index.ts

```diff
@@ -124,18 +124,24 @@ export const GNOSIS_FORUM_ROADTODECENT_LINK = 'https://forum.gnosis.io/t/gpv2-ro
 export const MEV_TOTAL = '606 Million'
 export const FLASHBOTS_LINK = 'https://explore.flashbots.net/'
 
-// 30 minutes
-export const GAS_PRICE_UPDATE_THRESHOLD = 30 * 60 * 1000
+//  10 seconds
+export const GAS_PRICE_UPDATE_THRESHOLD = 10 * 1000
 export const GAS_FEE_ENDPOINTS = {
-  [ChainId.MAINNET]: 'https://safe-relay.gnosis.io/api/v1/gas-station/',
+  [ChainId.MAINNET]: 'https://api.blocknative.com/gasprices/blockprices',
   // No ropsten = main
   // [ChainId.ROPSTEN]: 'https://safe-relay.gnosis.io/api/v1/gas-station/',
-  [ChainId.RINKEBY]: 'https://safe-relay.rinkeby.gnosis.io/api/v1/gas-station/',
-  [ChainId.GOERLI]: 'https://safe-relay.goerli.gnosis.io/api/v1/gas-station/',
+  [ChainId.RINKEBY]: 'https://api.blocknative.com/gasprices/blockprices', // not actually supported
+  [ChainId.GOERLI]: 'https://api.blocknative.com/gasprices/blockprices', // not actually supported
```

Comment:
Then remove/comment out?

Reply:
Can't completely remove because TS will complain because of chainIds